docs/architecture/adr-008-dCERT-group.md

```diff
@@ -0,0 +1,149 @@
+aggregate and relay input from a variety of stakeholders to the developers who
+are actively devising a patch to the software, this way sensitive information
+does not need to be publicly disclosed while some input from the community can
+still be gained. 
+
+Additionally, a special privilege is proposed for the dCERT group:
+the capacity to "circuit-break" (aka. temporarily disable)  a particular
+message path. 
+
+In the future it is foreseeable that the community may wish to expand the roles
+of dCERT with further responsibilities such as the capacity to "pre-approve" a
+security update on behalf of the community prior to a full community
+wide vote whereby the sensitive information would be revealed prior to a
+vulnerability being patched on the live network.  
+
+## Decision
+
+The dCERT group is proposed to include an implementation of a `SpecializationGroup`
+as defined in [ADR 007](./adr-007-specialization-groups.md). This will include the 
+implementation of: 
+ - continuous voting
+ - slashing due to breach of soft contract
+ - revoking a member due to breach of soft contract
+ - emergency disband of the entire dCERT group (ex. for colluding maliciously) 
+ - compensation stipend from the community pool or other means decided by
+   governance
+
+This system necessitates the following new parameters: 
+ - blockly stipend allowance per dCERT member 
+ - maximum number of dCERT members 
+ - required staked slashable tokens for each dCERT member 
+ - unbonding time for dCERT staked token 
+ - quorum for suspending a particular member 
+ - proposal wager for disbanding the dCERT group 
+
+These parameters are expected to be implemented through the param keeper such 
+that governance may change them at any given point. 
+
+### Continuous Voting Electionator
+
+An `Electionator` object is to be implemented with continuous voting is to be
+implemented with the following specifications:
+ - All delegation addresses may submit votes at any point which updates their 
+   preferred representation on the dCERT group. 
+ - Preferred representation may be arbitrarily split between addresses (ex. 50%
+   to John, 25% to Sally, 25% to Carol) 
+ - Addresses which control the greatest amount of preferred-representation are
+   eligible to join the dCERT group (up the _maximum number of dCERT members_)
+   - In the split situation where the dCERT group is full but a vying candidate 
+     has the same amount of vote as an existing dCERT member, the existing 
+     member should maintain its position. 
+ - In order for a new member to be added to the dCERT group they must 
+   send a transaction accepting their admission at which point the validity of
+   their admission is to be confirmed. 
+   - A sequence number is assigned when a member is added to dCERT group. 
+     If a member leaves the dCERT group and then enters back, a new sequence number
+     is assigned.  
+ - If the dCERT group is already full and new member is admitted, the existing
```

Comment:
Kind of a dup of the 2nd previous point. Maybe combine them?

Reply:
Good point. I merged these two together.